src/FileFormats/MOF/mof.1.2.schema.json

```diff
@@ -92,6 +92,13 @@
                         },
                         "set": {
                             "$ref": "#/definitions/scalar_sets"
+                        },
+                        "primal_start": {
+                            "type": "number"
+                        },
+                        "dual_start": {
+                            "description": "An initial scalar value for constraint duals that the optimizer may use to warm-start the solution process.",
```

Comment:
If we're going to add descriptions, we should add them also to `primal_start`, and we should update them in https://github.com/jump-dev/MathOptFormat/pull/22 as well.